app/frontend_api.py

```diff
@@ -1,3 +1,18 @@
+#!/usr/bin/python2.7
```

Comment:
Would you mind deleting this line? I know it's in a lot of the existing Python files, but I don't think it's necessary (at least not in general), and I'd rather not add more of them, especially since we're working on moving to Python 3.

Reply:
Acknowledged

Reply:
Why not change this to #!/usr/bin/env python?

Reply:
I don't think we need the line at all; this isn't a standalone file that would be run on its own.